snappy-dunits/src/test/scala/io/snappydata/dunit/streaming/TweetToRowsConverter.scala

```diff
@@ -2,15 +2,15 @@ package io.snappydata.dunit.streaming
 
 import twitter4j.Status
 
-import org.apache.spark.sql.catalyst.InternalRow
+import org.apache.spark.sql.Row
 import org.apache.spark.sql.streaming.StreamToRowsConverter
 import org.apache.spark.unsafe.types.UTF8String
 
 class TweetToRowsConverter extends StreamToRowsConverter with Serializable {
 
-  override def toRows(message: Any): Seq[InternalRow] = {
+  override def toRows(message: Any): Seq[Row] = {
     val status: Status = message.asInstanceOf[Status]
-    Seq(InternalRow.fromSeq(Seq(status.getId,
+    Seq(Row.fromSeq(Seq(status.getId,
```

Comment:
Catalyst layer will convert a String to UtfString internally. Why we needed to do it?